public/vendor/VERSIONS

```diff
@@ -58,3 +58,12 @@ Version: 4.3.0
 
 File(s): /vendor/assets/swagger-ui/
 Version: 3.0.4
+
```

Comment:
There is also the other file in `public/vendor` where libraries need to be listed (incl. which license)

Reply:
I see, fixed.